packages/eslint-plugin/rules/index.js

```diff
@@ -1,7 +1,7 @@
 /**
  * ESLint rules: custom rules for Site Kit.
  *
- * Site Kit by Google, Copyright 2021 Google LLC
```

Comment:
The copyright dates don't need to update when the file is changed. We may have done that in the past but we don't anymore. It doesn't need to be undone, just FYI.